CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 ### Changed
 
 - Expose `mock_display::MockDisplay` for use in tests, examples, etc in crates that pull embedded-graphics in as a dependency.
+- **(breaking)** #243 All methods on `DrawTarget` that handle drawing to/updating the display are now fallible and return `Result<(), Self::Error>` to allow for error handling in user code. This also affects `Drawable::draw()`, which now also returns a `Result<(), Self::Error>`.
```

Comment:
The description of the `Drawable::draw` return type is a bit misleading, because `Self::Error` doesn't refer to the `Drawable` but the `DrawTarget` in this case. This could just say that errors `Drawable::draw` propagates errors that occurred while drawing without explicitly mentioning the type.